crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -15,6 +15,11 @@ struct FragmentInput {
 };
 
 @fragment
+// NOTE: in the future, if we do depth writes, we'll also need to disable
```

Comment:
As far as I know we currently do write to the depth texture, is this going to be an issue? Unless I'm misunderstanding something.

Reply:
_If I understand it correctly_, the restriction is just for explicit writes from within the shader, e.g. for effects like parallax occlusion mapping, ray-marched, impostor textures with depth.. and not for fixed function depth writes coming from the rasterizer.